content/user/security/relocate_download_folder.md

```diff
@@ -29,19 +29,25 @@ To do this, you must:
 
 In Zen Cart 1.5.5 and higher: 
 
+<div class="indent">
```

Comment:
Instead of adding specific HTML markup here, does using Markdown's markup for indenting and syntax-highlighting work? 

Three back-ticks to start, and three more to end the section.
Optionally specify a language for syntax highlighting after the opening 3 backticks (none after the closing set).

Reply:
Example: /content/dev/code/notifiers.md

Reply:
That makes the whole block look like code.  I just wanted it to be indented.